core/src/main/java/tc/oc/pgm/blitz/BlitzMatchModule.java

```diff
@@ -123,7 +123,7 @@ public void handleSpawn(final ParticipantSpawnEvent event) {
                           lives == 1 ? "misc.life" : "misc.lives",
                           NamedTextColor.AQUA,
                           text(lives))),
-                  Title.Times.of(Duration.ZERO, fromTicks(60), fromTicks(20))));
+                  Title.Times.times(Duration.ZERO, fromTicks(60), fromTicks(20))));
```

Comment:
Were these breaking changes? Even if we update, I'd rather continue to use the deprecated methods to minimize the amount of breakage too shading among other things.

Reply:
Let's keep them for now then.

Reply:
I've tested all changes in the PR and this specifically does not appear to be a breaking change. The titles appear as configured. However I am uncertain how to test the switch to `GlobalTranslator.translator()` fully, though there appears to be no issues with translations showing correctly when a client is using a language other than English in my testing.